AssetLoader.cs

```diff
@@ -1,15 +1,17 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
+using System.Globalization;
+using System.Collections.Generic;
 using Microsoft.Xna.Framework.Audio;
 using Microsoft.Xna.Framework.Graphics;
-using Rampastring.Tools;
 using Microsoft.Xna.Framework.Content;
 using Microsoft.Xna.Framework.Media;
+using Color = Microsoft.Xna.Framework.Color;
 using SixLabors.ImageSharp;
 using SixLabors.ImageSharp.Formats.Png;
-using Color = Microsoft.Xna.Framework.Color;
-using System.Globalization;
+using SixLabors.ImageSharp.PixelFormats;
+using Rampastring.Tools;
+using SixLabors.ImageSharp.Formats;
```

Comment:
Please keep usings sorted in alphabetical order.

Reply:
Done by 63882dd8b85bf1acea447e6d46afcd3ad4ddb85d